examples/linear_model/plot_lasso_coordinate_descent_path.py

```diff
@@ -60,10 +64,10 @@
 
 plt.figure(2)
 ax = plt.gca()
-ax.set_color_cycle(2 * ['b', 'r', 'g', 'c', 'k'])
-l1 = plt.plot(-np.log10(alphas_lasso), coefs_lasso.T)
-l2 = plt.plot(-np.log10(alphas_positive_lasso), coefs_positive_lasso.T,
-              linestyle='--')
+l1 = [plt.plot(-np.log10(alphas_lasso), coef, c=next(colors))
+      for coef in coefs_lasso][-1]
+l2 = [plt.plot(-np.log10(alphas_positive_lasso), coef, linestyle='--', c=next(colors))
+      for coef in coefs_positive_lasso][-1]
```

Comment:
as you don't need the l1 and l2 and I would not use list comprehension but a plain for loop. it's also more readable.